backend/controller/InsightsController.js

```diff
@@ -49,3 +49,66 @@ exports.getMinutesListened = async (spotifyId, listenStats) => {
 
   return Math.floor(msListened / 1000 / 60);
 };
+
+exports.getArtistListens = async (spotifyId, listenStats) => {
+  // https://stackoverflow.com/questions/1296358/how-to-subtract-days-from-a-plain-date
+  const lastWeek = new Date();
+  lastWeek.setDate(lastWeek.getDate() - 7);
+  const lastWeekTimestamp = lastWeek.toISOString();
+
+  // of the form { trackId: # listens }
+  let listens = {};
```

Comment:
nit (I think it should be the same)
```suggestion
  const listens = {};
```